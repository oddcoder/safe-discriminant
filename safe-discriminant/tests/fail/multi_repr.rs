use safe_discriminant::Discriminant;

#[derive(Discriminant)]
#[repr(C, u8, i8)]
pub enum Foo {
    A = 0,
    B = 1,
}

fn main() {}
