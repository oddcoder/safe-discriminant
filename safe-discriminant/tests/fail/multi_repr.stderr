error[E0566]: conflicting representation hints
 --> tests/fail/multi_repr.rs:4:8
  |
4 | #[repr(C, u8, i8)]
  |        ^  ^^  ^^
  |
  = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
  = note: for more information, see issue #68585 <https://github.com/rust-lang/rust/issues/68585>
  = note: `#[deny(conflicting_repr_hints)]` on by default
